src/__tests__/test-service.ts

```diff
@@ -108,12 +108,12 @@ export class TestTigrisService {
 			reply.setStatus("created");
 			callback(undefined, reply);
 		},
-		// eslint-disable-next-line @typescript-eslint/no-empty-function
+		/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
why is this change necessary ?

Reply:
I am not sure if linting ran on the test files previously.

Due to indentation, the function definition is not in a single line anymore and as a result `eslint-disable-next-line` doesn't work anymore. This annotation is used when you want to disable a rule of a couple of lines.